powersimdata/design/generation/cost_curves.py

```diff
@@ -442,7 +442,7 @@ def plot_c1_vs_c2(
         raise ValueError(f"{gen_type} is not a valid generation type.")
 
     # Identify the load zones that correspond to the specified area and area_type
-    returned_zones = area_to_loadzone(grid, area, area_type)
+    returned_zones = area_to_loadzone(area, area_type)
```

Comment:
Similar commend as above: should this be `area_to_loadzone(grid.get_grid_model(), area, area_type)`?

Reply:
Done